docs/README.md

```diff
@@ -2,6 +2,7 @@
 
 ## Introduction
 * [README](../README.md). Overview of Avalon and its source code
+* [ARCHITECTURE](avalan-arch.pdf). Overview of Avalon architecture
```

Comment:
change `avalan-arch.pdf` to `avalon-arch.pdf`

Reply:
good catch! fixed